gslab_scons/release.py

```diff
@@ -100,14 +112,16 @@ def main(version = None,
                            provenance_path   = provenance_path)
 
 
-def inspect_repo():
+def inspect_repo(scons_local_path = None):
     '''Ensure the repo is ready for release.'''
-    if not _release_tools.up_to_date(mode = 'scons'):
+    if not _release_tools.up_to_date(mode = 'scons', scons_local_path = scons_local_path):
         raise ReleaseError('SCons targets not up to date.')  
     elif not _release_tools.up_to_date(mode = 'git'):
-        print "Warning: `scons` has run since your latest git commit.\n"
+        warn = "Warning: your git working tree is not clean.\n" \
+               + "End this process and run `git status` for more infomration."
+        print warn
         response = raw_input("Would you like to continue anyway? (y|n)\n")
-        if response in ['N', 'n']: 
+        if bool(re.match('no?', response, re.IGNORECASE)):
```

Comment:
I prefer the original version for readability. (see [xkcd](https://xkcd.com/1171/)).

Reply:
Sometimes it's nice to communicate in words (see [xkcd](https://xkcd.com/1306/))